drivers/gles3/storage/material_storage.cpp

```diff
@@ -1640,8 +1640,8 @@ MaterialStorage::MaterialStorage() {
 		actions.renames["NODE_POSITION_VIEW"] = "(model_matrix * scene_data.view_matrix)[3].xyz";
 
 		actions.renames["VIEW_INDEX"] = "ViewIndex";
-		actions.renames["VIEW_MONO_LEFT"] = "0";
-		actions.renames["VIEW_RIGHT"] = "1";
+		actions.renames["VIEW_MONO_LEFT"] = "uint(0)";
```

Comment:
Before I forget, @clayjohn @dsnopek this was an interesting thing I ran into on OpenGL. When we're rendering in multiview `ViewIndex` is replaced by `gl_ViewID_OVR`, else it was replaced by `0`.

That is working fine in our Vulkan implementation. In OpenGL it was giving me problems because `gl_ViewID_OVR` is a `uint` and for some reason it's typing `0` and `1` as `int` and I got a type error on it. Hence the cast to `uint`.

This feels to me like a settings issue, it should be able to cast constants to either `uint` or `int` automatically as needed.

Reply:
Note that I was testing with a shader in Godot that is actually using the ViewIndex variable.

Reply:
OpenGL doesn't like casting between int and uint, we are stuck being explicit everywhere unfortunately.